airqo-platform/AirQo-frontend
# Modification to the FOR YOU section

#### Summary of Changes (What does this PR do?)

- This PR adds Filters for KYA and Quiz in the KnowYourAirView widget. 

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

-    https://www.figma.com/file/agN9GOkAbhXfGDEXYCKwBu/Mobile-App?type=design&node-id=7193%3A41108&mode=design&t=OEn591JTP4jvoMdU-1   Confirm the designs 

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

#### Screenshots (optional)
